### ISSUE TITLE
macOS (.app.zip) support

### ISSUE THREAD

**Issue by keatongreve** [microsoft/fastlane-plugin-appcenter#35]
Trying to get this working for a macOS application we have, not much luck since I was getting an error related to acceptable file formats.

I have a naive solution [here](https://github.com/Microsoft/fastlane-plugin-appcenter/compare/master...keatongreve:COURT-AllowMacOS). If anyone has any feedback on this, I'm more than willing to open a PR for it.

edit: I'm just passing the path to the app.zip to the `:ipa` option since based on the source, it works the same. we probably should have a differently named option for consistency (though if anyone's ever used the `hockey` fastlane action, the option works the same way. [source](https://docs.fastlane.tools/actions/hockey/)).

**Comment by lumaxis** [comment]
@keatongreve I like this! Also thanks for bringing this up, we should definitely have better support for macOS apps 🙂 
What do you think about narrowing this down to `.app.zip` extensions? I'm slightly worried that loosening the check to allow all `.zip` files might lead to unwanted side effects of allowing unsupported files.

**Comment by keatongreve** [comment]
I definitely agree on that. I went with the lazy approach because `File.extname` doesn't really work for `.<original_extension>.zip`.
```
// before you run irb, `touch fake.app.zip` in shell's current directory
2.5.0 :001 > File.extname("fake.app.zip")
 => ".zip" 
```

I was looking to see if there was already a standard lib call for this case, but no luck so far and I ran out of free time for this. I'm happy to make it work specifically for `.app.zip` but i didn't want to re-invent the wheel.
